src/OmniSharp.MSBuild/ProjectSystem.cs

```diff
@@ -157,7 +159,7 @@ IEnumerable<string> CollectFunc(string slnproj)
 
         private IEnumerable<string> GetProjectPathsFromSolutionDefault(string solutionFilePath)
         {
-            _logger.LogInformation($"Detecting projects in '{solutionFilePath}'.");
+            _logger.LogInformation($"Detecting projects in '{solutionFilePath}' .");
```

Comment:
Did you mean to add a space before the period here?

Reply:
It's mistake, I fixed it.